src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -1175,13 +1175,12 @@ def visit_Assign(self, node: ast.Assign) -> list:
             else:
                 raise GTScriptSyntaxError(message="Invalid target in assignment.", loc=target)
 
-            axes = self.fields[t.id].axes
             par_axes_names = [axis.name for axis in gt_ir.Domain.LatLonGrid().parallel_axes]
             if self.iteration_order == gt_ir.IterationOrder.PARALLEL:
                 par_axes_names.append(gt_ir.Domain.LatLonGrid().sequential_axis.name)
-            if set(par_axes_names) - set(axes):
+            if set(par_axes_names) - set(self.fields[t.id].axes):
                 raise GTScriptSyntaxError(
-                    message=f"Cannot assign to a field unless all parallel axes are present: '{par_axes_names}'.",
+                    message=f"Cannot assign to field '{node.targets[0].id}' as all parallel axes '{par_axes_names}' are not present.",
```

Comment:
`axes = self.fields[t.id].axes` seems useful to keep around. Was there a reason to remove it?

Reply:
It was only used in one place.

Reply:
Oh, you're right. My bad...